bin/live/pycbc_live_supervise_collated_trigger_fits

```diff
@@ -656,7 +667,11 @@ def supervise_collation_fits_dq(args, day_dt, day_str):
         )
     # Store the locations of files needed for the statistic
     stat_files = []
-    for ifo in config_opts['control']['ifos'].split():
+
+    all_ifos = controls['ifos'].split()
+    active_ifos = get_active_ifos(merged_triggers)
+    active_ifos = [ifo for ifo in active_ifos if ifo in all_ifos]
```

Comment:
I think here we are looking at removing any IFOs which are in the trigger file but not in the config file? That may be worth commenting, as I got confused for a minute

Reply:
Added a comment